### ISSUE TITLE
Inconsistency in attribute name in IdentifierKeyValuePair class

### ISSUE THREAD

**Issue by markus-ap** [admin-shell-io/aas-specs#61]
```json
  "specificAssetIds": [
     {
        "key": "EquipmentID",
        "value": "538fd1b3-f99f-4a52-9c75-72e9fa921270",
        "subjectId": {
           "keys": [
              {
                 "type": "GlobalReference",
                 "value": "http://customer.com/Systems/ERP/012",
                 "idType": "IRI"
              }
           ]
        }
     }
```             
In [jsonExample.json](https://github.com/admin-shell-io/aas-specs/blob/master/schemas/json/examples/jsonExample.json) the attribute`subjectId` is used, whereas the [Part 1 document](https://www.plattform-i40.de/IP/Redaktion/EN/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part1_V3.pdf?__blob=publicationFile&v=5) specifies `externalSubjectId`. The same inconsistency is seen in JSON examples within the Part 1 document.

I am not sure which name should be the correct one, as I can't find any reference to a change of this attribute, other than it being introduced alongside `IdentifierKeyValuePair`.

**Comment by BirgitBoss** [comment]
externalSubjectId is the correct name in V3.0RC01

**Comment by Manu3756** [comment]
JSON-Schema will be fixed with V3.0RC02.
